geode-core/src/distributedTest/java/org/apache/geode/cache30/ReconnectWithCacheXMLDUnitTest.java

```diff
@@ -118,7 +117,7 @@ public void beforeMembershipFailure(String reason, Throwable cause) {
       }
     });
     MembershipManagerHelper.crashDistributedSystem(cache.getDistributedSystem());
-    assertTrue(membershipFailed.get());
+    await().untilAsserted(() -> membershipFailed.get());
```

Comment:
This line won't do anything as written. It needs to either be:
```
await().until(() -> membershipFailed.get());
```
Or:
```
await().untilAsserted(() -> assertThat(membershipFailed.get()).isTrue());
```

Reply:
done